executor/show_test.go

```diff
@@ -16,13 +16,13 @@ package executor_test
 import (
 	"context"
 	"fmt"
-
```

Comment:
Please do not remove this line, keep the sys libs separate with the 3rd party libs.

Reply:
Adding the blank line will cause check_dev to fail.

Reply:
Why check_dev could pass in master branch? I think you've added a blank line with some white spaces, try to keep it as it used to be.

Reply:
I fixed it.